node_modules/oae-tenants/tests/test-tenants.js

```diff
@@ -218,6 +241,65 @@ describe('Tenants', function() {
         });
 
         /**
+         * Test that verifies a tenant can be looked up by an email domain match
+         */
+        it('verify get tenant by email domain', function(callback) {
+            var exactMatchTenantAlias = TenantsTestUtil.generateTestTenantAlias();
+            var exactMatchTenantDescription = TestsUtil.generateRandomText();
+            var exactMatchTenantHost = TenantsTestUtil.generateTestTenantHost();
+            var exactMatchTenantOpts = {'emailDomain': TenantsTestUtil.generateTestTenantHost()};
+
+            var suffixTenantAlias = TenantsTestUtil.generateTestTenantAlias();
+            var suffixTenantDescription = TestsUtil.generateRandomText();
+            var suffixTenantHost = TenantsTestUtil.generateTestTenantHost();
+            var suffixTenantOpts = {'emailDomain': util.format('*.%s', exactMatchTenantOpts.emailDomain)};
+
```

Comment:
Can we try and use some real values here?

For example:
 * cam.ac.uk
 * admin.cam.ac.uk
 * sports.cam.ac.uk
 * uis.cam.ac.uk
 *  ...

should all resolve to the same tenant configured with `*cam.ac.uk`